src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -19,6 +19,11 @@
      * changes on the returned list will not affect the person's internal tags.
      */
     UniqueTagList getTags();
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public String getPrintableString(Printable... printables);
```

Comment:
You have not implemented this method to print what printables that's getting passed in.